pkg/cmd/get.go

```diff
@@ -62,6 +65,7 @@ func NewGetCmd() *cobra.Command {
 	o.kubeFlags.AddFlags(cmd.Flags())
 	cmd.Flags().BoolVarP(&o.AllNamespaces, "all-namespaces", "A", o.AllNamespaces, "If present, list the requested object(s) across all namespaces. Namespace in current context is ignored even if specified with --namespace.")
 	cmd.Flags().StringVar(&o.KubeArchiveHost, "kubearchive-host", o.KubeArchiveHost, fmt.Sprintf("Host where the KubeArchive API Server is listening. Defaults to '%s'", o.KubeArchiveHost))
+	cmd.Flags().StringVarP(&o.Output, "output", "o", "", "Output format. One of: (json, yaml).")
```

Comment:
I'm wondering if the default should have a name, feels weird to have "defaults to an empty string" or "do not pass it to print the resources as a table". What do you think about having three: table, json or yaml, being "table" the deafult.

Reply:
You shouldn't be able to specify `-o` by itself.

> "do not pass it to print the resources as a table" 

This is the way `kubectl` works.  Don't pass `-o` and you get the table.

Reply:
For reference, this is the piece of code in the `kubectl get` module that includes the output option: https://github.com/kubernetes/kubectl/blob/master/pkg/cmd/get/get_flags.go#L155

The option uses this `JSONYamlPrintFlags` defined [here](https://github.com/kubernetes/cli-runtime/blob/master/pkg/genericclioptions/json_yaml_flags.go) in which the options are just `json` and `yaml` (no table) so, I agree with Greg that we should stick with the same design.

Actually, I'm wondering if we could use exactly the same flags from the library `k8s.io/cli-runtime/pkg/genericclioptions` what do you think?